rust/arrow/src/datatypes.rs

```diff
@@ -219,9 +218,9 @@ pub trait ArrowPrimitiveType: 'static {
     /// the corresponding Arrow data type of this primitive type.
     const DATA_TYPE: DataType;
 
-    /// Returns the bit width of this primitive type.
-    fn get_bit_width() -> usize {
-        size_of::<Self::Native>() * 8
+    /// Returns the byte width of this primitive type.
+    fn get_byte_width() -> usize {
+        size_of::<Self::Native>()
```

Comment:
I like this change 👍

Reply:
Yeah, this `* 8` and `/ 8` was always a bit funny to me, mostly because I was not sure whether the compiler was able to realize about it.